test/powershell/Language/Classes/Scripting.Classes.BasicParsing.Tests.ps1

```diff
@@ -573,9 +573,9 @@ Describe 'Hidden Members Test ' -Tags "CI" {
 
         # Formatting should not include hidden members by default
         $tableOutput = $instance | Format-Table -HideTableHeaders -AutoSize | Out-String
-        It "Table formatting should not have included hidden member hiddenZ - should contain 10" { $tableOutput.Contains(10) | Should -BeTrue}
-        It "Table formatting should not have included hidden member hiddenZ- should contain 12" { $tableOutput.Contains(12) | Should -BeTrue}
-        It "Table formatting should not have included hidden member hiddenZ - should not contain 42" { $tableOutput.Contains(42) | Should -BeFalse}
+        It "Table formatting should not have included hidden member hiddenZ - should contain 10" { $tableOutput.Contains("10") | Should -BeTrue}
```

Comment:
Perhaps we can update this test similar to what we have in the Format-Table.tests.ps1 which is instead of these 3 checks, we can have:

```powershell
# hiddenZ should not show up in table
$expectedTable = @"

visibleX visibleY
-------- --------
      10       12



"@

$tableOutput = $instance | Format-Table -AutoSize | Out-String
$tableOutput | Should -Be -Exactly $expectedTable
```

Reply:
Updating this test

Reply:
I created  tracking Issue #6738 to consider this in more detail.